teraranger_evo_node:
  ros__parameters:
    device_name: 'TR_evo_3m'
    serial_port: '/dev/ttyACM0'
    frame_id: 'sensor_frame' 
